cli/hse_cli.c

```diff
@@ -164,19 +165,23 @@ struct cli_cmd        cli_hse_kvs_commands[] = {
  *    hse storage profile
  */
 static cli_cmd_func_t cli_hse_storage_add;
-#ifdef HSE_EXPERIMENTAL
-static cli_cmd_func_t cli_hse_storage_info;
-#endif
 static cli_cmd_func_t cli_hse_storage_profile;
 struct cli_cmd        cli_hse_storage_commands[] = {
     { "add", "Add a new media class storage to an existing offline KVDB", cli_hse_storage_add, 0 },
-#ifdef HSE_EXPERIMENTAL
-    { "info", "Display storage stats for a KVDB", cli_hse_storage_info, 0 },
-#endif
     { "profile", "Profile storage path to determine throttle policy", cli_hse_storage_profile, 0 },
     { 0 },
 };
 
+/*****************************************************************
```

Comment:
hmmm... Why do we need a separate sub-command for mclass? Can we report the per-media class stats in "hse storage info" output in verbose mode?

Reply:
I felt like this matched previous UX.

hse_kvdb_kvs_create() -> hse kvs create
hse_kvdb_mclass_info_get() -> hse mclass info

Reply:
Not a big fan of "hse mclass info". I would retain "hse storage info" to dump aggregated stats and add a verbose option to "storage info" to dump the per-media class stats.

IMHO, adding a new "mclass" sub-command is a bit confusing when we already have a "storage" sub-command to retrieve anything about KVDB storage.

Reply:
@smoyergh could use your opinion here.

Reply:
I agree with @nabeelmmd.